lib_eio_linux/tests/test.ml

```diff
@@ -19,7 +23,7 @@ let read_one_byte ~sw r =
     )
 
 let test_poll_add () =
-  Eio_linux.run @@ fun _stdenv ->
+  Eio_linux.run ~fallback:skip (fun _stdenv ->
```

Comment:
Still no need to make the change from `@@` to `(`

Reply:
Thank you @patricoferris -- I  have gone through the documentation, To get it correct, I used `Alcotest.skip`. Something like this 

```
let skip_io_uring () = 
    Alcotest.skip "Skipping test: io_uring not available in Docker";;
``` 
However, when i use this, i encounter this error: `Alcotest.skip "Skipping test: io_uring not available in Docker";;
Error: This expression has type string but an expression was expected of type unit`

Is there something i am doing wrong? I would appreciate help.

Reply:
@create2000 the type of `skip` is 

```ocaml
val skip : unit -> 'a 
```

It doesn't expect a string as the first argument but a value of type `unit` (i.e `()`).  You could perhaps log something with `Eio.traceln` and then `Alcotest.skip ()`. Not that this is only for the Alcotest-based Linux tests.

Reply:
@patricoferris Thank you for the clarification. I have modified it and logged a message using the `Eio.tracein` which was displayed in the generated file after running the file. 

This is the new modification: 

```
let skip_io_uring () =
  Eio.traceln "Skipping test: io_uring not available in Docker";
  Alcotest.skip ()
```

```
let test_poll_add () =
  Eio_linux.run ~fallback:(skip_io_uring()) 
```